src/bundle/Resources/public/js/scripts/admin.content.tree.js

```diff
@@ -5,10 +5,11 @@
     const CLASS_BTN_CONTENT_TREE_EXPANDED = 'ez-btn--content-tree-expanded';
     const token = doc.querySelector('meta[name="CSRF-Token"]').content;
     const siteaccess = doc.querySelector('meta[name="SiteAccess"]').content;
+    const userId = doc.querySelector('meta[name="UserId"]').content;
```

Comment:
Idea: Could we introduce `eZ.getCurrentUserId()`/`eZ.helper.getCurrentUserId()` to avoid code duplication? I feel this line will be in all *.js which somehow needs current user id